src/features/orders/OrderTabPanel.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import Tabs from '@material-ui/core/Tabs'
+import Tab from '@material-ui/core/Tab'
+import { types } from './reducer'
+import Box from '@material-ui/core/Box'
+import TabPanel from './TabPanel'
+import OrderInfo from './OrderInfo'
+import CustomerInfo from './CustomerInfo'
+import ItemsInfo from './DetailItems'
+import { makeStyles, Theme } from '@material-ui/core/styles'
+
+const useStyles = makeStyles((theme: Theme) => ({
+    root: {
+        margin: '0 1em',
+    },
+    panelWrapper: {
+        boxShadow: theme.shadows[4],
+        padding: '1em',
+        backgroundColor: theme.palette.common.white,
+        borderRadius: '1em',
+        borderColor: theme.palette.background.default,
+        borderWidth: '5px',
+        borderStyle: 'solid',
+    },
+    indicator: {
+        backgroundColor: 'transparent',
+    },
+    tabs: {
+        position: 'relative',
+        top: '5px',
+        width: '95%',
+        margin: 'auto',
+    },
+    tab: {
+        backgroundColor: theme.palette.common.white,
+        borderTopRightRadius: '1em',
+        borderTopLeftRadius: '1em',
+        borderBottomWidth: '5px',
+        borderBottomColor: theme.palette.background.default,
+        borderBottomStyle: 'solid',
+        zIndex: 1,
+    },
+    tabSelected: {
+        borderColor: theme.palette.background.default,
+        borderWidth: '5px',
+        borderStyle: 'solid',
+        borderBottomWidth: '0',
+        zIndex: 2,
+        boxShadow: theme.shadows[4],
+    },
+}));
+
+export interface OrderTabPanelProps {
+    tab: number;
+    onChangeTab: (id: number) => void;
+    data: types.Record,
+}
+
+export default function OrderTabPanel({
+    tab,
+    onChangeTab,
+    data,
+}: OrderTabPanelProps ) {
+    const classes = useStyles();
+
+    return (
+        <Box className={classes.root}>
+            <Tabs 
+                classes={{ indicator: classes.indicator, root: classes.tabs }}
+                value={tab} 
+                onChange={(event, value) => onChangeTab(value as number)}
+                textColor='primary'
+                variant='fullWidth'
+            >
+                <Tab 
+                    classes={{ 
```

Comment:
all tabs have the same classes and disableRipple props., Either define these props on the top and pass it into each tab using spread operator or define your custom local Tab component (located in this file) where these props set by default